sdks/python/setup.py

```diff
@@ -557,6 +557,20 @@ def get_portability_package_data():
               'torch',
               'transformers',
           ],
+          'p313_ml_test': [
```

Comment:
Rather than adding a p3XX_ml_test version for every supported python version, could we rename `p312_ml_test` to something like `ml_basic_test`? I think `ml_test` is just this + dill and tensorflow-transform FWIW

Reply:
I know this is technically breaking, but I would be surprised if anyone is depending on this directly (and we could call this out in CHANGES)